en/docs/administer/monitoring-server-health.md

```diff
@@ -28,39 +28,54 @@ The **Carbon Health Check API** can be used to check the health of WSO2 Identity
 
 ### Configuring the API
 
-This feature is disabled by default. To enable the API, set the
-`         <Enable>        ` property in the
-`         health-check-config.xml        ` file to **true**.  
+This feature is disabled by default. To enable the API, add following
+configuration to `deployment.toml` file in the [IS_HOME]/repository/conf
+location.
+
+```toml
+[carbon_health_check]
+enable= true
+```
 
 !!! tip
     
     If the feature has not been enabled successfully, a request to the API
     will only return a 200 OK response.
     
+There are two health-checkers that will get enabled when you enable
+health checkers using above global configuration. 
+    - DataSource Health Checker 
+    - User Store Health Checker
 
-**Sample health-check-config.xml file**
+**DataSource Health Checker**
 
-``` java
-<CarbonHealthCheckConfigs>
-    <Enable>true</Enable>
-    <HealthCheckers>
-        <HealthChecker name="DataSourceHealthChecker" orderId="97" enable="true">
-            <!--<Property name="monitored.datasources">jdbc/WSO2CarbonDB,jdbc/WSO2MetricsDB,jdbc/WSO2UMDB</Property>-->
-            <Property name="pool.usage.limit.percentage">80</Property>
-        </HealthChecker>
-        <HealthChecker name="SuperTenantUSHealthChecker" orderId="98" enable="true">
-            <!--<Property name="monitored.user.stores">primary,sec</Property>-->
-        </HealthChecker>
-    </HealthCheckers>
-</CarbonHealthCheckConfigs>
+Gives you the status about the datasources available in the system. You
+can configure the datasources to be monitored as below in
+`deployment.toml` file.
+
+```toml
+[carbon_health_check.health_checker.data_source_health_checker.properties]
+'monitored.datasources' = "jdbc/WSO2CarbonDB,jdbc/WSO2UM_DB,jdbc/SHARED_DB"
 ```
 
--   A health checker can be enabled or disabled using the
-    **`           enable          `** attribute.
--   The execution order in which the health checkers are executes can be
-    configured using the **`           orderId          `** attribute.
--   The properties configured under each health checker will be
-    available for each heath checker at runtime.
+To provide a indicate about datasource connection pool usage, use
+following configuration in `deployment.toml` file.
```

Comment:
Shall we modify this to "To provide an **indication** about the datasource connection pool usage, use **the** following configuration..."